java/client/src/main/java/glide/api/models/commands/ConditionalChange.java

```diff
@@ -0,0 +1,28 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.GeospatialIndicesBaseCommands;
+import java.util.Map;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * An optional condition to the {@link GeospatialIndicesBaseCommands#geoadd(String, Map,
+ * GeoAddOptions)} command.
+ */
+@RequiredArgsConstructor
+@Getter
+public enum ConditionalChange {
```

Comment:
Delete it from `SetOptions` and refer from there

Reply:
py client also reuses that

Reply:
I will do this in a different PR Zadd also uses it.

Reply:
please don't forget